op-chain-ops/script/script.go

```diff
@@ -386,6 +386,17 @@ func (h *Host) GetNonce(addr common.Address) uint64 {
 	return h.state.GetNonce(addr)
 }
 
+func (h *Host) ImportState(allocs *foundry.ForgeAllocs) {
```

Comment:
Might be worth a doc comment that it does not erase existing state when importing